PowerUp/app/src/main/res/layout/intro_screen_1.xml

```diff
@@ -0,0 +1,14 @@
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools" android:layout_width="match_parent"
+    android:layout_height="match_parent" tools:context="powerup.systers.com/IntroActivity2"
+    android:background="@drawable/background">
+
+    <TextView
```

Comment:
In the previous version you had this text inside a speech bubble in an image. Why don't you still have this image as a background and put this TextView on top of it?